### ISSUE TITLE
Socket count:0

### ISSUE THREAD

**Issue by Racaway** [gkbrk/slowloris#15]
When executing your code on a local network, the socket count gives as output 0. We attack from a Mac to another Mac(Host pc), but it appears that no sockets are actually being made. Do you have an idea about what is causing this and how to fix it?

Thanks In advance

**Comment by gkbrk** [comment]
@Racaway Can the computers normally send packets to each other? Some wifi routers don't allow clients to communicate with each other. A good way to try this is to load a page from the HTTP server on the other mac.

Other than that, which HTTP server are you using on the host PC? It usually depends on the server that is being used.

**Comment by EmanueleLillo** [comment]
Exactly, Apache is the most vulnerable, because It creates a new thread every client that connects. If you're using other WebServers or Reverse Proxies like nginx (event based), the attack won't work.

**Comment by spradha1** [comment]
Hi all. I am having the same problem with Apache. I am using a the most recent version which is why it is not working and my Ubuntu does not seem to like install the older vulnerable versions of Apache. Is there a way to configure the recent version to make the attack possible? Thanks in advance!
